src/main/resources/templates/petbio/add.html

```diff
@@ -0,0 +1,76 @@
+
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org/">
+<head th:replace="fragments :: head"></head>
+<body class="container">
+
+<header th:replace="fragments :: header"></header>
+
+
+<form method="post">
+    <div class="form-group">
+        <label>Name
+            <input th:field="${petBio.petName}" class="form-control">
+        </label>
+        <p class="error" th:errors="${petBio.petName}" style="color:red";></p>
+    </div>
+    <div class="form-group">
+        <label>Age
+            <input th:field="${petBio.petAge}" class="form-control">
+        </label>
+        <p class="error" th:errors="${petBio.petAge}" style="color:red";></p>
+    </div>
+
+    <div class="form-group">
+        <label>Species
+            <input th:field="${petBio.petSpecies}" class="form-control">
+        </label>
+        <p class="error" th:errors="${petBio.petSpecies}" style="color:red";></p>
+    </div>
+
+    <div class="form-group">
+        <label>Breed
+            <input th:field="${petBio.petBreed}" class="form-control">
+        </label>
+        <p class="error" th:errors="${petBio.petBreed}" style="color:red";></p>
+    </div>
+
+    <div class="form-group">
+        <label>Gender
+            <select th:field="${petBio.petGender}" class="form-control">
+                <option th:value="Male">Male</option>
+                <option th:value="Female">Female</option>
```

Comment:
Do we need to add another option? I don't know if the sex of animals is non-binary.

Reply:
I added non-binary

Reply:
I added non-binary